web/app/src/components/editors/forwarder/otlp.tsx

```diff
@@ -0,0 +1,98 @@
+import Divider from '@mui/material/Divider'
+import TextField from '@mui/material/TextField'
+
+import { OtlpForwarderModel } from '@/lib/models/forwarder/otlp'
+
+type OtlpForwarderEditorProps = {
+  config: OtlpForwarderModel
+  onConfigChange: (config: OtlpForwarderModel) => void
+}
+
+export const OtlpForwarderEditor = ({
+  config,
+  onConfigChange,
+}: OtlpForwarderEditorProps) => {
+  const headers = config.headers || {}
+
+  return (
+    <div className="flex flex-col gap-4">
+      <TextField
+        id="input:forwarder.otlp.endpoint"
+        label="Endpoint"
+        value={config.endpoint}
+        onChange={(e) =>
+          onConfigChange({
+            ...config,
+            endpoint: e.target.value,
+            headers: headers, // Ensure headers is included
+          })
+        }
+        type="text"
+        variant="outlined"
+        required
+        className="w-full"
+        placeholder="http://localhost:4318/v1/logs"
+      />
+
+      <Divider />
+
+      <div className="flex flex-col gap-2">
+        <h3 className="text-lg font-semibold">Headers</h3>
```

Comment:
Have a look at the `KeyValueEditor` component in `web/app/src/components/form/kv-editor.tsx`